.github/workflows/release.yml

```diff
@@ -1,26 +1,46 @@
 ---
 name: Release
 on:
-  push:
-    tags:
-      - 'v*'
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release tag'
+        required: true
+      previous-tag:
+        description: 'Previous release tag'
+        required: true
 jobs:
   release:
     name: Release
+    if: ${{ github.repository == 'shipwright-io/build' }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: write # To be able to update releases
```

Comment:
GitHub has tightened the security model around actions. We now need to grant explicit permission to create releases.